resources/builder.inc.sh

```diff
@@ -2140,6 +2140,33 @@ builder_is_target_excluded_by_platform() {
   return 1
 }
 
+# Returns 0 if the script is running on a CI server (TeamCity or GitHub Actions)
+builder_is_running_on_ci() {
```

Comment:
This might cause confusion in usage/maintenance with 
https://github.com/keymanapp/keyman/blob/3634dd170225f8316333cdae46c5f3e365a47992/resources/build/build-utils-ci.inc.sh#L10-L19 (which is in build-utils-ci.inc.sh)

Reply:
Ooh @darcywong00 good catch! Testing the same outcome a different way. 

I would lean towards keeping the existing function and not adding builder_is_running_on_ci. Perhaps cross-reference the various functions in the documentation.

Reply:
Good catch, thanks @darcywong00!

This raises two questions:
- should the three `build_is_ci_*` functions be moved from `resources/build/build-utils-ci.inc.sh` to `resources/builder.inc.sh` for better disoverability?
- do we have to document these three functions?

Reply:
Probably yes, build-utils-ci.inc.sh is better. It is good to document all the 'public' functions.